examples/wasm-demo-cairo1/README.md

```diff
@@ -0,0 +1,49 @@
+# Demo of `cairo-vm` on WebAssembly
+
+While cairo-vm is compatible with WebAssembly, it doesn't implement any bindings to it.
+Instead, create a new WebAssembly crate with cairo-vm as a dependency and implement the required functionality there.
+
+Since mimalloc is not automatically compilable to WebAssembly, the cairo-vm dependency should disable the default features, which will in turn disable mimalloc.
+
+A working example is provided in this repository.
+
+## Dependencies
+
+To compile and run the example you need:
+
+- a Cairo 0 compiler
+- the _wasm-pack_ crate
+- some HTTP server (for example: the `live-server` npm module)
+
+> **Note**
+> The first two dependencies can be installed via the repository's installation script (see ["Installation script"](../../README.md#installation-script))
+
+## Building
+
+To build the example, first compile your Cairo 1.x program:
+
+```sh
+cairo1/bin/cairo-compile -r ./bitwise.cairo  bitwise.sierra
```

Comment:
I think you should show the commands relative to the current dir, so that it's easier to follow. For example:
```suggestion
../../cairo1/bin/cairo-compile -r bitwise.cairo  bitwise.sierra
```
Also, the above command will fail as `bitwise.cairo` is actually in `src/bitwise.cairo` (you may want to move it or update the command)